.eslintrc.js

```diff
@@ -13,15 +23,18 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120 }],
     'vue/attribute-hyphenation': 'off',
+    'vue/multi-word-component-names': 'off',
```

Comment:
This is to fix line 258 in `Dashboard.vue`.

Reference: https://stackoverflow.com/questions/71205264/component-name-temp-should-always-be-multi-word-vue-multi-word-component-names

Reply:
I agree -- turn this one off (not even a warning).

Please make sure all 3 UIs have the same ESLint presets and rule overrides.

Reply:
^^ Create new tickets if needed.